polaris-react/src/components/Bleed/Bleed.stories.tsx

```diff
@@ -35,7 +35,7 @@ export function Default() {
 
 export function WithVerticalDirection() {
   return (
-    <Box background="surface" padding="4">
+    <Box background="surface-success" padding="4">
```

Comment:
Maybe we match these styles to the [styleguide](https://www.figma.com/file/5oEi1wm73KhGLN0VS02oYm/Layout-components?node-id=2765%3A32014&t=WnCODY49P5Sq7HFA-4) examples? I think it's clearer what's happening with just the border vs a bg color